templates/uber-development.ini.erb

```diff
@@ -74,6 +74,14 @@ shirt_sales_enabled = <%= @shirt_sales_enabled %>
 show_affiliates = <%= @show_affiliates %>
 student_discount = <%= @student_discount %>
 
```

Comment:
Since ``hotel_req_hours`` has a default value provided, we probably don't want/need to wrap this in an if-statement and could instead just say ``hotel_req_hours = <%= @hotel_req_hours %>``, right?